lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -3386,6 +3389,26 @@ class SPIRVJointMatrixINTELWorkItemInst : public SPIRVJointMatrixINTELInstBase {
 _SPIRV_OP(JointMatrixGetElementCoord, true, 5)
 #undef _SPIRV_OP
 
+class SPIRVCooperativeMatrixKHRInstBase : public SPIRVInstTemplateBase {
+protected:
+  std::optional<ExtensionID> getRequiredExtension() const override {
+    return ExtensionID::SPV_KHR_cooperative_matrix;
+  }
+  SPIRVCapVec getRequiredCapability() const override {
+    return getVec(CapabilityCooperativeMatrixKHR);
+  }
+};
+
+#define _SPIRV_OP(x, ...)                                                      \
+  typedef SPIRVInstTemplate<SPIRVCooperativeMatrixKHRInstBase, Op##x,          \
+                            __VA_ARGS__>                                       \
+      SPIRV##x;
+_SPIRV_OP(CooperativeMatrixLoadKHR, true, 5, true)
+_SPIRV_OP(CooperativeMatrixStoreKHR, false, 4, true)
```

Comment:
If I'm not mistaken how the macro works - these lines should be looking like this, as MatrixLoad and MatrixStore in KHR extension have 2 optional parameters.
```suggestion
_SPIRV_OP(CooperativeMatrixLoadKHR, true, 4, true, 2)
_SPIRV_OP(CooperativeMatrixStoreKHR, false, 3, true, 2)
```